src/theory/bags/normal_form.cpp

```diff
@@ -0,0 +1,27 @@
+/*************************                                                    */
+/*! \file normal_form.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Mudathir Mohamed
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **/
+
+#include "normal_form.h"
+
+namespace CVC4 {
+namespace theory {
+namespace bags {
+
+bool NormalForm::checkNormalConstant(TNode n)
+{
+  // TODO(#223): complete this function
```

Comment:
Nit: this is technically a projects issue, not to be confused with the main issues, maybe write: `projects#223`.

Reply:
done along with other TODOs as well